### ISSUE TITLE
NPE : parentcontext null at install of defaultcontextinteraction

### ISSUE THREAD

**Issue by klmp200** [sarl/sarl#682]
```
java.lang.NullPointerException
	at io.janusproject.kernel.bic.DefaultContextInteractionsSkill.install(DefaultContextInteractionsSkill.java:99)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at io.janusproject.kernel.bic.AgentLifeCycleSupport.installSkills(AgentLifeCycleSupport.java:178)
	at io.janusproject.kernel.bic.AgentLifeCycleSupport.agentSpawned(AgentLifeCycleSupport.java:90)
	at io.janusproject.kernel.services.jdk.spawn.StandardSpawnService.fireAgentSpawnedInAgent(StandardSpawnService.java:274)
	at io.janusproject.kernel.services.jdk.spawn.StandardSpawnService.lambda$0(StandardSpawnService.java:209)
	at io.janusproject.kernel.services.jdk.spawn.StandardSpawnService.spawn(StandardSpawnService.java:216)
	at io.janusproject.kernel.bic.LifecycleSkill.spawnInContext(LifecycleSkill.java:110)
	at io.sarl.demos.sierpinski.agents.Fractal.multiplication(Fractal.java:123)
	at io.sarl.demos.sierpinski.agents.Fractal.$behaviorUnit$Multiply$3(Fractal.java:168)
	at io.sarl.demos.sierpinski.agents.Fractal.lambda$3(Fractal.java:344)
	at io.janusproject.services.executor.JanusRunnable.runWithEarlyExitSupport(JanusRunnable.java:69)
	at io.janusproject.services.executor.JanusRunnable.run(JanusRunnable.java:61)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

Please add an assert in the constructor to ensure context is not null

**Comment by gallandarakhneorg** [comment]
The source of the problem is inside `StandardBuiltinCapacitiesProvider`:
```java
final DefaultContextInteractionsSkill interactionSkill = new DefaultContextInteractionsSkill(agent,
					this.contextRepository.getContext(agent.getParentID()));
```

The context repository replies a `null` value for the parent context.

A possible meaning is: the agent is created into a super-agent, but the super agent's inner context is destroyed before this agent has finished to be created.
Hopefully, a super agent can be killed only if it has no member. But, the sub-agent is not yet created? So that the super-agent accept to be destroyed.

This possible invalid behavior should be tested, and guards should be added in order to avoid this issue.

**Comment by gallandarakhneorg** [comment]
Should be reproduced and fixed after Issue #751.
